lib/travis/services/find_admin.rb

```diff
@@ -10,11 +10,11 @@ class FindAdmin < Base
       include Travis::Logging
 
       register :find_admin
+      NUM_CANDIDATES = 15
```

Comment:
Thank you for the explanation @aakritigupta!

Maybe it's an edge case but I was wondering if we could perhaps make `NUM_CANDIDATES`'s value configurable?

Furthermore, maybe we could add a log when a valid Admin isn't found in 15 candidates (or the value in the configuration)?

I think we could detect this edge case this way and it could be fixed by just updating `NUM_CANDIDATES`'s value in the config rather than modifying the code.

I could be totally yak shaving here. It's just that I'm always wary when seeing hardcoded values in code.

Just my 2¢ and I'll leave you with deciding if what I propose is legitimate and worthy of doing.

Thank you!